CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+## 10.0.0
+
+Another major release! We have some great improvements in this version but also have breaking changes. Please review the following to see if you need to change anything to upgrade `zapier-platform-core` to v10.
+
+First, Zapier integrations that depend on the new Core v10 **will run using Node.js 12**. You can install Node.js 12 using `nvm`.
+
+Second, **`z.request` now always calls `response.throwForStatus`** via a middleware by default. You no longer need to call `response.throwForStatus` after `z.request`, the built-in middleware will do that for you. See [Error Response Handling](https://github.com/zapier/zapier-platform/blob/master/packages/cli/README.md#error-response-handling) for details.
+
+Third, **`response.throwForStatus` now only throws an error if the status code is between 400 and 600**. Before v10, it throws for status >= 300. So if your code rely on that old behavior, you should check `response.status` explicitly instead of using `response.throwForStatus`.
+
+Fourth, we now **parse JSON and form-encoded response body by default**. The parsed object is available as `response.data` (`response.json` will be still available for JSON body but less preferable). Before v10, we only parse JSON for [manual requests](https://github.com/zapier/zapier-platform/blob/master/packages/cli/README.md#manual-http-requests); parse JSON and form-encoded body for [shorthand requests](https://github.com/zapier/zapier-platform/blob/master/packages/cli/README.md#shorthand-http-requests). This change could be breaking if you have an `afterResponse` that modifies `response.content`, with the expectation for shorthand requests to pick up on that. In which case, you'll have to replace `response.content = JSON.stringify(parsedOrTransformed)` with `response.data = parsedOrTransformed`.
```

Comment:
> Before v10, we only parse JSON for [manual requests]

Two things about that line: 

1. should be `we only parsed` JSON for manual requests
2. also, it was undocumented, but it's not accurate. We always tried to parse json out and put it on `response.json`, even on "manual" `z.request`. See discussion [here](https://zapier.slack.com/archives/CA6FTB140/p1588787498041100?thread_ts=1588785216.036500&cid=CA6FTB140).

So since the "official story" is that they used to have to parse it themselves and no longer have to, this line is fine (besides the typo)

Reply:
Also, I really like the line:

> This change could be breaking if you have an `afterResponse` that modifies `response.content`, with the expectation for shorthand requests to pick up on that. In which case, you'll have to replace `response.content = JSON.stringify(parsedOrTransformed)` with `response.data = parsedOrTransformed`.

because it clearly says what action they need to take and if it applies to them.